ui/admin/app/components/form/field/list-wrapper/select/index.hbs

```diff
@@ -0,0 +1,85 @@
+<Hds::Table class='list-wrapper-field' name={{@name}}>
+
+  <:body as |B|>
+
+    {{#each this.options as |select index|}}
+      <B.Tr>
+        <B.Td>
+          <Hds::Form::Select::Base
+            @value={{select.value}}
+            @width={{@width}}
+            disabled={{@disabled}}
+            aria-label={{t 'titles.value'}}
+            {{on 'change' (set-from-event select 'value')}}
+            as |F|
+          >
+            <F.Options>
+              {{#each @selectOptions as |selectOption|}}
+                <option
+                  value={{selectOption}}
+                  selected={{eq select.value selectOption}}
+                >
+                  {{selectOption}}
+                </option>
+              {{/each}}
+            </F.Options>
+          </Hds::Form::Select::Base>
+        </B.Td>
+
+        <B.Td>
+          {{#if this.removeOptionByIndex}}
+            <Hds::Button
+              data-test-remove-option-button={{select.value}}
+              @text={{t 'actions.remove'}}
+              @color='critical'
+              @icon='trash'
+              @isIconOnly={{true}}
+              disabled={{@disabled}}
+              {{on 'click' (fn this.removeOptionByIndex index)}}
+            />
+          {{/if}}
+        </B.Td>
+      </B.Tr>
+    {{/each}}
+
+    <B.Tr>
+      <B.Td>
+        <Hds::Form::Select::Base
+          @value={{this.newOptionValue}}
+          @width={{@width}}
+          disabled={{@disabled}}
+          aria-label={{t 'titles.value'}}
+          {{on 'change' (set-from-event this 'newOptionValue')}}
+          as |F|
+        >
+
+          <F.Options>
+            <option disabled={{@disabled}} hidden selected value=''>
```

Comment:
This should always be disabled

Reply:
@ZedLi the reason I used `form.disabled` to manage that `option` is because when you edit the form and you add a new algorithm to the list the next table row doesn't show the 'Choose an Option' default it shows the first default in the array.. Of course the data isn't saved, and its disabled, I thought it would look funny if the user saw that dropdown.  Any ideas?
<img width="1296" alt="image" src="https://github.com/hashicorp/boundary-ui/assets/24277002/12a11865-8532-47ea-92c3-ac495319c8b2">

Reply:
Makes sense, I think that's fine then